Controller/Adminhtml/Importcategory/Save.php

```diff
@@ -80,136 +80,161 @@ public function execute()
+                                        $cateItem->setUrlKey($_url_key);
+                                        $cateItem->setStoreId($storeId);
+                                        $cateItem->save();
+                                        
+                                        if ($cateItem->getId()) {
+                                            $existingCategories[$cateItem->getId()] = $cateItem->getName();
+                                            $existingCategoriesPath[$cateItem->getId()] = $cateItem->getPath();
+                                            $existingCategoriesPathName[$cateItem->getId()] = $categoryItem[$categoriesKey];
                                         }
                                     }
                                 }
-                            } elseif (isset($categoryidkey) && ($categoryidkey!='' || $categoryidkey===0) && $storedata!='') {
+                            } elseif (isset($categoryIdKey) && ($categoryIdKey != '' || $categoryIdKey === 0) && $storeData != '') {
                                 //update categories
-                                $catemodel = $this->_categoryFactory->create();
-                                if (!$singlestoremode && isset($_stores[$categoryitem[$storedata]])) {
-                                        $catemodel->setStoreId($_stores[$categoryitem[$storedata]]);
+                                $cateModel = $this->_categoryFactory->create();
+                                if (!$singleStoreMode && isset($_stores[$categoryItem[$storeData]])) {
+                                    $cateModel->setStoreId($_stores[$categoryItem[$storeData]]);
                                 } else {
-                                    $catemodel->setStoreId($storeId);
+                                    $cateModel->setStoreId($storeId);
                                 }
-                                $cateitem = $catemodel->load($categoryitem[$categoryidkey]);
-                                $nocategoryfound = true;
-                                if ($cateitem->getId()) {
-                                    $nocategoryfound = false;
-                                    $attributesetid = $cateitem->getAttributeSetId();
-                                    $_parentid = $cateitem->getParentId();
+                                $cateItem = $cateModel->load($categoryItem[$categoryIdKey]);
+                                $noCategoryFound = true;
+                                if ($cateItem->getId()) {
+                                    $noCategoryFound = false;
+                                    $attributesetid = $cateItem->getAttributeSetId();
+                                    $_parentId = $cateItem->getParentId();
                                     foreach ($cat_data as $key => $value) {
                                         if (!in_array($key, ['url_key','category_id','url_path','path','level','children_count','full_path'])) {
```

Comment:
if you are doing refactoring then make this line match 120 characters limit also.


Remove older usage of declaring private/protected methods in PHP
e.g. change `$_parentId` to `$parentId`